docs/Panorama/event-system.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: Event System
+parent: Panorama
+nav_order: 100
+---
+
+# Event System
+Panorama uses an event system to communicate between the engine and panels but also between panels. They can be defined and subscribed to via Javascript.
+
+## Defining Events
+```
+**TODO** What is the difference between these two functions
+```
+You can define your own global events within the ``scripts/util/event-definition.js`` file by calling the ``$.DefineEvent`` and ``$.DefinePanelEvent`` functions.
+
+### Example
+```js
+$.DefineEvent("LoadTutorialMap", 0);  // Event "LoadTutorialMap" with no arguments
+$.DefineEvent("LoadStoryMap", 1);     // Event "LoadStoryMap" with one argument
+```
+
+## Subscribing to Events
+Any panel can subscribe to an event by calling ``$.RegisterForUnhandledEvent`` or ``$.RegisterEventHandler`` within JavaScript. These functions will be called, when the event is fired. Some events may include additional parameters, which will be forwarded to your function.
+
+#### $.RegisterForUnhandledEvent(``EVENT_NAME``, ``CALLBACK``)
+This function can be used to subscribe to events.
```

Comment:
Notably, this will only get fired if none of the handlers registered with "RegisterEventHandler" returned true (or i guess a truthy value in a JS setting)